AdyenNetworking/APIClient/Request.swift

```diff
@@ -45,7 +45,7 @@ public protocol Request: Encodable {
     
     /// :nodoc:
     /// The HTTP headers.
-    var headers: [String: String] { get }
+    var headers: [String: String] { get set }
```

Comment:
let me know if this is ok with folks, it removes the need for a MutableRequest on our side at least, which is just used to allow setting these headers